### PR TITLE
implemented mailchip sign up

### PR DESCRIPTION
**Pull request by makayi**
I have implemented signing up for the mailchimp newsletter.  The user's name is being posted in the  mail chimp.  One issue I need to solve is adding the tag "newsletter for each news signup to catergorize them into tags"

### CODE REVIEW

**Comment by malgamves** [comment]
image
Hey, i tried filling it in and it was stuck displaying the text "sending".. what is the expected behavior?

File: src/components/MailChimp.js [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+        this.setState({ status: "success" });
+      }
+    });
+  }
+
+  render() {
+    const {
+      fields,
+      styles,
+      className,
```
Comment (malgamves) [inline src/components/MailChimp.js]: Hey this variable wasn't used LTGM is giving us errors.

File: src/components/MailChimp.js [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+      buttonClassName,
```
Comment (malgamves) [inline src/components/MailChimp.js]: Hey this variable wasn't used LTGM is giving us errors.

File: src/components/MailChimp.js [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+      inputClassName
```
Comment (malgamves) [inline src/components/MailChimp.js]: Hey this variable wasn't used LTGM is giving us errors.

**Comment by makayi** [comment]
I will give it a look. Thanks

**Comment by malgamves** [comment]
Also, looking at #154 could we have the newsletter form have titles for inputs and not placeholder texts?

**Comment by makayi** [comment]
Sure thing.

**Comment by k9uma** [comment]
I would like to merge this then open issues for any other pending changes.. is that fine? @malgamves

**Comment by malgamves** [comment]
Yes, fine with me

**Pull request merged**